new-nextjs-client/pages/index.module.scss

```diff
@@ -1,6 +1,19 @@
 .container {
   padding: 0 2rem;
   background: $main-background;
+  display: grid;
+  grid-template-rows: 7rem auto;
+
+  .topLayout {
+    display: flex;
+    justify-content: space-between;
+    .addYear {
+      width: 14rem;
```

Comment:
The width should be defined in AddYearDropdown component.
Define this at the top level div container for AddYearDropdown. 
index.tsx should not have knowledge of the width of AddYearDropdown component.